February-11-2015-Updates.md

```diff
@@ -0,0 +1,52 @@
+
+
+## Retiring Camper News
+- Over the past year, Camper News has been a modest success. But so has our subreddit, which allows for commenting, moderation, and many other features. So we are shifting to Reddit for news sharing. 
+- We recommend creating a Reddit account if you don't already have one and [joining our subreddit](https://www.reddit.com/r/freecodecamp). You can now share links with our entire community there, as well as your thoughts and questions.
+- We will keep Camper News live for a while, but we're removing the link to it from our navbar, and removing the challenge that introduced it.
+
+
+## Updating challenges
+- Many campers asked that we create an easier, HTML-focused project to proceed "Build a Personal Portfolio Webpage". So we've created a new project that can be completed using only HTML and Bootstrap called "Build a Tribute Page".
+- We also resequenced our front end projects. Here's how they're now structured:
+  - Basic Front End Projects
+    - Build a Tribute Page
+    - Build a Personal Portfolio Webpage
+  - Intermediate Front End Projects
+    - Build a Random Quote Machine
+    - Show the Local Weather
+    - Build a Wikipedia Viewer
+    - Use the Twitch.tv JSON API
+  - Advanced Front End Projects
+    - Build a JavaScript Calculator
+    - Build a Pomodoro Clock
+    - Build a Tic Tac Toe Game
+    - Build a Simon Game
+- You'll note that we've removed the "Build a Camper News Viewer" project since we're phasing out Camper News. If you've previously completed that challenge, the "Build a Tribute Page" should already be marked completed for you on your map.
+- Also, for "Build a Tic Tac Toe Game", you no longer need to make it unbeatable. (You certainly can if you want - [here's how](http://neverstopbuilding.com/minimax). Although doing so is a useful exercise, writing an AI is a departure from the project's focus, which is front end development.
+- We added three new optional challenges that help you join our:
+  - [subreddit](https://reddit.com/r/freecodecamp
+  - join our Medium publication
+  - join our Twitch.tv channel
+
+## Shop
+- We have a new t-shirt campaign going on until February 15. Hurry up and buy one of these awesome Free Code Camp t-shirts: [click here for our US campaign](https://teespring.com/free-code-camp-feb) | [click here for our EU campaign](https://teespring.com/free-code-camp-t-shirt-eu-shop)
+- We also have Free Code Camp laptop stickers. You get two for only $5, with free shipping anywhere on earth!
+
+## Coming soon
+- We will launch our Spanish challenges in the next few days.
+- We plan to release optional React, Sass, and D3 challenges in March to teach you these technologies. But don't wait - you can complete the projects with the help of these projects' official documenation - 16 campers have already done just that.
```

Comment:
What is this mainly doing for us?  Why are we switching to reddit?   Are we going to keep github?